cypress/integration/User Management/BJ_00_InstructorEdit.js

```diff
@@ -0,0 +1,130 @@
+describe("Fills out form", () => {
+    before(() => {
+        cy.fixture("users.json").then(({instructor, instructor2, inviteInstructorConfirmation}) => {
```

Comment:
refactor `instructor1` and `instructor2` to be more descriptive: `original_instructor_data` and `edited_instructor_data`. I had to go through the entire file to understand what was `instructor2`

refact`inviteInstructorConfirmation` - this should be a noun, not a verb. Is this the response to the instructor confirmation? be more descriptive with the name.